tests/click_tests/common.py

```diff
@@ -64,21 +65,23 @@ def tap_to_confirm(debug: "DebugLink") -> LayoutContent:
     elif debug.layout_type is LayoutType.Caesar:
         return debug.read_layout()
     elif debug.layout_type is LayoutType.Delizia:
-        return debug.click(buttons.TAP_TO_CONFIRM)
+        debug.click(buttons.TAP_TO_CONFIRM)
+        return debug.read_layout()
```

Comment:
please simplify this function :) it's returning `read_layout` in all codepaths, the only thing we need to do here is "if delizia then tap", maybe we don't even need to return the layout? (but i don't know the usage sites, maybe we do)

Reply:
Good catch, thanks!
It is indeed used only in `test_autolock.py`, so we can indeed simplify it :)
```
$ git grep tap_to_confirm
click_tests/common.py:def tap_to_confirm(debug: "DebugLink") -> LayoutContent:
click_tests/test_autolock.py:from .common import go_next, tap_to_confirm, unlock_gesture
click_tests/test_autolock.py:        layout = tap_to_confirm(debug)
```

Reply:
Fixed in 26ab041f4.